app/views/supervisor/dashboard/index.html.slim

```diff
@@ -0,0 +1,9 @@
+h1 = "Hi #{current_user.name}!"
+h2 This is your brandnew Supervisor Dashboard - with love from Team Cheesy
+
+hr
+
+- for t in supervised_teams.each do
+  h4 = link_to t.name, t
+  p = "Your latest check was #{ time_ago_in_words(t.last_checked_at) } ago."
```

Comment:
@F3PiX I just noticed the stated time is incorrect. You thought it had something to do with the default timezones right? Should we do anything about it or is going to be too complicated since it's an app-wide thing?